examples/e2e/.eslintrc.yml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
so, by default we use [StandardJS](https://github.com/feross/standard)... this filed should not be needed.